packages/react-devtools-shared/src/devtools/views/root.css

```diff
@@ -1,280 +1,25 @@
-  --dark-color-scheduling-profiler-react-render-selected:#64A9F5;
-  --dark-color-scheduling-profiler-react-render-hover:#2683E2;
-  --dark-color-scheduling-profiler-react-commit: #ff718e;
-  --dark-color-scheduling-profiler-react-commit-selected:#FF5277;
-  --dark-color-scheduling-profiler-react-commit-hover:#ed0030;
-  --dark-color-scheduling-profiler-react-layout-effects:#c88ff0;
-  --dark-color-scheduling-profiler-react-layout-effects-selected:#934FC1;
-  --dark-color-scheduling-profiler-react-layout-effects-hover:#601593;
-  --dark-color-scheduling-profiler-react-passive-effects:#c88ff0;
-  --dark-color-scheduling-profiler-react-passive-effects-selected:#934FC1;
-  --dark-color-scheduling-profiler-react-passive-effects-hover:#601593;
-  --dark-color-scheduling-profiler-react-schedule: #9fc3f3;
-  --dark-color-scheduling-profiler-react-schedule-hover:#2683E2;
-  --dark-color-scheduling-profiler-react-schedule-cascading:#ff718e;
-  --dark-color-scheduling-profiler-react-schedule-cascading-hover:#ed0030;
-  --dark-color-scheduling-profiler-react-suspend: #a6e59f;
-  --dark-color-scheduling-profiler-react-suspend-hover:#13bc00;
-  --dark-color-scheduling-profiler-text-color: #000000;
-  --dark-color-scheduling-profiler-react-work-border:#ffffff;
   --dark-color-scroll-thumb: #afb3b9;
   --dark-color-scroll-track: #313640;
-  --dark-color-search-match: yellow;
-  --dark-color-search-match-current: #f7923b;
-  --dark-color-selected-tree-highlight-active: rgba(23, 143, 185, 0.15);
-  --dark-color-selected-tree-highlight-inactive: rgba(255, 255, 255, 0.05);
-  --dark-color-shadow: rgba(0, 0, 0, 0.5);
-  --dark-color-tab-selected-border: #178fb9;
-  --dark-color-text: #ffffff;
-  --dark-color-text-invalid: #ff8080;
-  --dark-color-text-selected: #ffffff;
-  --dark-color-toggle-background-invalid: #fc3a4b;
-  --dark-color-toggle-background-on: #178fb9;
-  --dark-color-toggle-background-off: #777d88;
-  --dark-color-toggle-text: #ffffff;
   --dark-color-tooltip-background: rgba(255, 255, 255, 0.95);
   --dark-color-tooltip-text: #000000;
 
   /* Font smoothing */
-  --light-font-smoothing: auto;
-  --dark-font-smoothing: antialiased;
   --font-smoothing: auto;
 
   /* Compact density */
-  --compact-font-size-monospace-small: 9px;
-  --compact-font-size-monospace-normal: 11px;
-  --compact-font-size-monospace-large: 15px;
-  --compact-font-size-sans-small: 10px;
-  --compact-font-size-sans-normal: 12px;
-  --compact-font-size-sans-large: 14px;
   --compact-line-height-data: 18px;
   --compact-root-font-size: 16px;
 
   /* Comfortable density */
-  --comfortable-font-size-monospace-small: 10px;
-  --comfortable-font-size-monospace-normal: 13px;
-  --comfortable-font-size-monospace-large: 17px;
-  --comfortable-font-size-sans-small: 12px;
-  --comfortable-font-size-sans-normal: 14px;
-  --comfortable-font-size-sans-large: 16px;
   --comfortable-line-height-data: 22px;
   --comfortable-root-font-size: 20px;
```

Comment:
Could use inline comments explaining why we leave each of the specific styles here. For example, I think I know why you've left `--comfortable-line-height-data` and `--compact-line-height-data` but I'm not sure about root-font-size.

Reply:
Actually on second thought, since we've pulled these values out of CSS into a JavaScript file, let's just export them so we can avoid hacky stuff like this?

https://github.com/facebook/react/blob/9f88b5355ba21e06e0a3c4442c5199de354ed1cf/packages/react-devtools-shared/src/constants.js#L67-L85

Instead we could do something like this:
```diff
diff --git a/packages/react-devtools-shared/src/constants.js b/packages/react-devtools-shared/src/constants.js
index 10750fa7d7..2d7f614c92 100644
--- a/packages/react-devtools-shared/src/constants.js
+++ b/packages/react-devtools-shared/src/constants.js
@@ -57,31 +57,10 @@ export const REACT_DEVTOOLS_WORKPLACE_URL =
 
 // HACK
 //
-// Extracting during build time avoids a temporarily invalid state for the inline target.
 // Sometimes the inline target is rendered before root styles are applied,
 // which would result in e.g. NaN itemSize being passed to react-window list.
-//
-let COMFORTABLE_LINE_HEIGHT;
-let COMPACT_LINE_HEIGHT;
-
-try {
-  // $FlowFixMe
-  const rawStyleString = require('!!raw-loader!react-devtools-shared/src/devtools/views/root.css')
-    .default;
-
-  const extractVar = varName => {
-    const regExp = new RegExp(`${varName}: ([0-9]+)`);
-    const match = rawStyleString.match(regExp);
-    return parseInt(match[1], 10);
-  };
-
-  COMFORTABLE_LINE_HEIGHT = extractVar('comfortable-line-height-data');
-  COMPACT_LINE_HEIGHT = extractVar('compact-line-height-data');
-} catch (error) {
-  // We can't use the Webpack loader syntax in the context of Jest,
-  // so tests need some reasonably meaningful fallback value.
-  COMFORTABLE_LINE_HEIGHT = 15;
-  COMPACT_LINE_HEIGHT = 10;
-}
+const {styles} = require ('react-devtools-shared/src/devtools/views/ThemeProvider');
+const COMFORTABLE_LINE_HEIGHT = parseInt(styles.comfortable['--line-height-data'], 10);
+const COMPACT_LINE_HEIGHT = parseInt(styles.compact['--line-height-data'], 10);
 
 export {COMFORTABLE_LINE_HEIGHT, COMPACT_LINE_HEIGHT};
diff --git a/packages/react-devtools-shared/src/devtools/views/ThemeProvider.js b/packages/react-devtools-shared/src/devtools/views/ThemeProvider.js
index a93076d30c..ffd3e29140 100644
--- a/packages/react-devtools-shared/src/devtools/views/ThemeProvider.js
+++ b/packages/react-devtools-shared/src/devtools/views/ThemeProvider.js
@@ -12,7 +12,7 @@ import {useContext, useMemo} from 'react';
 import {SettingsContext} from './Settings/SettingsContext';
 import type {Theme, DisplayDensity} from './Settings/SettingsContext';
 
-const styles: {[style: Theme | DisplayDensity]: any} = {
+export const styles: {[style: Theme | DisplayDensity]: any} = {
   light: {
     '--color-attribute-name': '#ef6632',
     '--color-attribute-name-not-editable': '#23272f',
```

Reply:
Yes that's a better approach and saves quite a lines of code

Reply:
Sure 👍🏼 If we're not using them, why not?